### PR TITLE
Tooltips: sort by count and fix render issue

### LINKED ISSUE #2620: Order metrics shown on tooltip
## Is your feature request related to a problem?

This tooltip is hard to follow:

Screenshot from 2020-12-02 10-53-19

## Describe the solution you'd like

Order the tooltip by volume somehow.

One potential ordering - take the last data point, order by that.

## Additional context

Not sure why versions which are at 0 every datapoint even show up.

Came up during #2560

### LINKED ISSUE #8014: Insights tooltip rendering problems
## Bug description

If there's lots of lines the hover pane becomes not readable:
Screen Shot 2022-01-12 at 23 58 27

The ordering is hard to interpret:
Screen Shot 2022-01-12 at 23 57 54

^ I need to closely look at the table to know yellow is 1.29.1

## Expected behavior

For ordering we sort them based on the value or color the values if we want to keep them alphabetical as they are now so it's easily visually detectable which is which.

## How to reproduce

1. https://app.posthog.com/insights/DFGqpUoM?events=%5B%7B%22id%22%3A%22instance%20status%20report%22%2C%22math%22%3A%22dau%22%2C%22name%22%3A%22instance%20status%20report%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22properties%22%3A%5B%7B%22key%22%3A%22realm%22%2C%22type%22%3A%22event%22%2C%22value%22%3A%5B%22cloud%22%5D%2C%22operator%22%3A%22is_not%22%7D%5D%2C%22custom_name%22%3A%22self-hosted%22%7D%5D&actions=%5B%5D&display=ActionsLineGraph&insight=TRENDS&interval=week&breakdown=posthog_version&date_from=-30d&new_entity=%5B%5D&properties=%5B%7B%22key%22%3A%22realm%22%2C%22type%22%3A%22event%22%2C%22value%22%3A%5B%22cloud%22%5D%2C%22operator%22%3A%22is_not%22%7D%5D&breakdown_type=event&filter_test_accounts=true (too many items to show)
2. https://app.posthog.com/insights/QEsYvzvb?events=%5B%7B%22id%22%3A%22organization%20usage%20report%22%2C%22math%22%3A%22dau%22%2C%22name%22%3A%22organization%20usage%20report%22%2C%22type%22%3A%22events%22%2C%22order%22%3A0%2C%22properties%22%3A%5B%5D%2C%22custom_name%22%3A%22self-hosted%20unique%20users%22%7D%5D&actions=%5B%5D&display=ActionsLineGraph&insight=TRENDS&interval=day&breakdown=posthog_version&date_from=-90d&new_entity=%5B%5D&properties=%5B%7B%22key%22%3A%22realm%22%2C%22type%22%3A%22event%22%2C%22value%22%3A%5B%22cloud%22%5D%2C%22operator%22%3A%22is_not%22%7D%5D&breakdown_type=event&filter_test_accounts=true (ordering confusing to interpret)

## Environment

- [x] PostHog Cloud
- [ ] self-hosted PostHog (ClickHouse-based), version/commit: _please provide_
- [ ] self-hosted PostHog (Postgres-based, legacy), version/commit: _please provide_

### PR DESCRIPTION
**Pull request by alexkim205**
## Changes

Fixes #2620
Fixes #8014

_wow look at that ~6K issue number difference_

Screen Shot 2022-01-13 at 1 29 07 PM

Screen Shot 2022-01-13 at 1 29 02 PM

## How did you test this code?

Stylistic changes

### CODE REVIEW

**Review by macobo — COMMENTED** [review]
If we're sorting by count, I'd resolve ties alphabetically instead.

Will leave the code portion of the review to marius :)

**Review by paolodamico — CHANGES_REQUESTED** [review]
This is definitely an improvement and I wouldn't be opposed to merging. However, I don't think this is the best solution. Consider this extreme case, tooltips are impossible to read. I would add an upper bound on how wide/tall (depending on row/column) the tooltip can be, and ellipsis the text there (maybe 2-3 rows). Everything else lgtm!

**Comment by alexkim205** [comment]
Thanks for the review, this should fix it to 2 rows.

Screen Shot 2022-01-18 at 12 21 14 PM
